packages/mdc-toolbar/README.md

```diff
@@ -0,0 +1,111 @@
+# MDC Toolbar
+
+MDC Toolbar provides an `mdc-toolbar` class for easily styling a RTL-aware
+toolbar. The toolbar scrolls with content by default, but supports fixed on
+top as well.
+
+
+## Installation
+
+```
+npm install --save @material/toolbar
+```
+
+
+## Usage
+
+Wrap the items with `mdc-toolbar` class in following way:
+
+```html
+<div class="mdc-toolbar">
+  <div class="mdc-toolbar__section">...</div>
+</div>
+```
+
+### Fix to top
+
+By default, toolbar scroll with the content, you can add `mdc-toolbar_fixed` to
```

Comment:
A few grammatical errors here; I believe this should be "By default, toolbars scroll with the page content. To keep the toolbar fixed to the top of the screen, add an `mdc-toolbar--fixed` class to the toolbar element".

I also think it's a good idea to show an HTML demonstrating this, similar to above.